### PR TITLE
Tpc geometry fix

### PR DESCRIPTION
**Pull request by hupereir**
[comment]: <> (Please tell us something about this pull request)
This pull request moves all the Tpc Geometry initialization relevant to ACTS (via MakeActsGeometry) from PHG4TpcElectronDrift to PHG4TpcDetector. This allows to have the TPC geometry properly initialized when running e.g. on PRDFs, some thing that the current implementation prevents, since PHG4TpcElectronDrift runs only for simulations. 

There is still some duplicated code between PHG4TpcDetector and PHG4TpcPadPlaneReadout, because the later uses some internally built constants to handle dead areas between regions and sectors. Ultimately this should be changed to use only what's stored in PHG4TpcCylinderGeom. This will be the subject of future commits. 

I have double checked that the content of PHG4TpcCylinderGeom is identical between before and after the change.

## Types of changes
[comment]: <> ( What types of changes does your code introduce? Put an `x` in all the boxes that apply: )
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work for users)
- [ ] Requiring change in macros repository (Please provide links to the macros pull request in the last section)
- [x] I am a member of [GitHub organization of sPHENIX Collaboration](https://github.com/orgs/sPHENIX-Collaboration/people), EIC, or ECCE (contact Chris Pinkenburg to join)

## What kind of change does this PR introduce? (Bug fix, feature, ...)

[comment]: <> ( What does this PR do? Linking to talk in software meeting encouraged )

## TODOs (if applicable)

[comment]: <> ( In case this is a draft PR, e.g. for running checks using Jenkins, please make the pull request as a draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/  )

### CODE REVIEW

**Comment by osbornjd** [comment]
There seems to be some issues in R1 of the TPC, did some parameter get overlooked?

https://nbviewer.jupyter.org/github/sPHENIX-Collaboration/QA-gallery/blob/jenkins-sPHENIX-test-tracking-low-occupancy-qa-3698-test-tracking_Event500_Sum30/QA-Tpc.ipynb

Or does this need to be run in conjunction with https://github.com/sPHENIX-Collaboration/macros/pull/646?

File: simulation/g4simulation/g4tpc/PHG4TpcPadPlane.cc [hunk 1]
```diff
@@ -20,6 +20,9 @@ PHG4TpcPadPlane::PHG4TpcPadPlane(const std::string &name)
 
 int PHG4TpcPadPlane::InitRun(PHCompositeNode *topNode)
 {
+  
+  std::cout << "PHG4TpcPadPlane::InitRun" << std::endl;
```
Comment (osbornjd) [inline simulation/g4simulation/g4tpc/PHG4TpcPadPlane.cc:24]: Extra print statement

**Review by osbornjd — COMMENTED** [review]
Looks fine to me other than the issues in R1 and a few leftover print statements, maybe there is just some parameter that is not being loaded properly

**Comment by hupereir** [comment]
> There seems to be some issues in R1 of the TPC, did some parameter get overlooked?
> 
> https://nbviewer.jupyter.org/github/sPHENIX-Collaboration/QA-gallery/blob/jenkins-sPHENIX-test-tracking-low-occupancy-qa-3698-test-tracking_Event500_Sum30/QA-Tpc.ipynb
> 
> Or does this need to be run in conjunction with [sPHENIX-Collaboration/macros#646](https://github.com/sPHENIX-Collaboration/macros/pull/646)?

My understanding is that it should run without, but I might be wrong. 
Some debugging is in order, definitly. (so far I have checked only that the cylinderGeom object were identical, not the result of PHG4TpcElectronDrift. I'll do just that today. 

Thanks for going through the code !

**Comment by hupereir** [comment]
All right. That should fix that. Let's see.

**Comment by osbornjd** [comment]
Interesting that the other regions didn't also bomb out given their max radii were missing also...

**Pull request merged**